### PR TITLE
VGC/BSS: Update with Series 1

### PR DESCRIPTION
**Pull request by DaWoblefet**
Announced on Pokemon Home. All the Paradox mons and Treasures of Ruin banned. Currently assuming Paldea dex (e.g. Charizard, Quagsire); there's no ruleset to actually test on cart atm, so that may be updated pending further research later. Pokemon Home does not show non-Paldea Dex Pokemon as legal.

Calling it "VGC 2023" because it runs into the actual VGC season now. Unfortunately, they've continued with their practice of Series-style formats, so I've continued that naming convention here.

### CODE REVIEW

File: data/rulesets.ts [hunk 1]
```diff
@@ -388,6 +388,22 @@ export const Rulesets: {[k: string]: FormatData} = {
 			}
 		},
 	},
+	paldeapokedex: {
+		effectType: 'ValidatorRule',
+		name: 'Paldea Pokedex',
+		desc: "Only allows Pok&eacute;mon native to the Paldea region (SV)",
+		banlist: ['Raichu-Alola', 'Weezing-Base'],
```
[the remark this answers is not in the document]
Comment (DaWoblefet) [reply data/rulesets.ts:395]: And Tauros-Base, good catch!

File: config/formats.ts [hunk 2]
```diff
@@ -153,11 +158,16 @@ export const Formats: FormatList = [
...
 
 		mod: 'gen9',
 		gameType: 'doubles',
-		ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Min Source Gen = 9', 'VGC Timer'],
+		ruleset: ['Flat Rules', '!! Adjust Level = 50', 'Paldea Pokedex', 'Min Source Gen = 9', 'VGC Timer'],
+		banlist: [
+			'Great Tusk', 'Brute Bonnet', 'Sandy Shocks', 'Scream Tail', 'Flutter Mane', 'Slither Wing', 'Roaring Moon',
+			'Iron Treads', 'Iron Moth', 'Iron Hands', 'Iron Jugulis', 'Iron Thorns', 'Iron Bundle', 'Iron Valiant',
+			'Ting-Lu', 'Chienpao', 'Wo-Chien', 'Chi-Yu',
+		],
```
Comment (KrisXV) [inline config/formats.ts:170]: Wouldn't the proper route be to give them the Sub-Legendary tag in `pokedex.ts`?

File: data/rulesets.ts [hunk 3]
```diff
@@ -388,6 +388,22 @@ export const Rulesets: {[k: string]: FormatData} = {
...
+			];
+			const species = this.dex.species.get(set.species || set.name);
+			if (!paldeaDex.includes(species.baseSpecies) && !paldeaDex.includes(species.name) &&
+				!this.ruleTable.has('+' + species.id)) {
```
Comment (KrisXV) [inline data/rulesets.ts:402]: I know you're just copying from other stuff but I'm pretty sure this doesn't actually unban the pokemon

File: config/formats.ts (hunk 2 above)
Comment (DaWoblefet) [reply config/formats.ts:170]: Are the Paradox mons classified Sub-Legendary? If not, it doesn't really change a whole lot.

**Pull request merged**